src/main/java/com/dev/cinema/model/dto/UserRequestDto.java

```diff
@@ -0,0 +1,39 @@
+package com.dev.cinema.model.dto;
+
+public class UserRequestDto {
+    private String email;
+    private String login;
+    private String password;
```

Comment:
let's have the two fields here that you use in AuthenticationController, and later in AuthenticationService